prime-router/src/main/kotlin/metadata/LivdMappers.kt

```diff
@@ -96,17 +96,54 @@ class LIVDLookupMapper : Mapper {
             filters.notEqualsIgnoreCase(LivdTableColumns.PROCESSING_MODE_CODE.colName, testProcessingModeCode)
         }
 
+        val testPerformedCode = values.firstOrNull {
```

Comment:
This can all be replaced by fixing line 88 to equalsIgnoreCase.  That is the bug I introduced.  I thought I testes this in my unit tests, so maybe we need to look at that too.